frontend/libs/studio-components/src/components/StudioTableRemotePagination/StudioTableRemotePagination.stories.tsx

```diff
@@ -42,32 +43,41 @@ export const Preview: Story = (args) => {
   const [pageSize, setPageSize] = useState<number>(5);
 
   const { handleSorting, sortedRows } = useTableSorting(rows, { enable: true });
-
   const rowsToRender = getRowsToRender(currentPage, pageSize, sortedRows || rows);
-  const totalPages = Math.ceil(rows.length / pageSize);
 
-  if (!rowsToRender.length && (sortedRows.length || rows.length)) {
+  const totalRows = rows.length;
+  const totalPages = Math.ceil(totalRows / pageSize);
+
+  // Fallback to page 1 if the current page is out of bounds
+  if (!rowsToRender.length && totalRows) {
     setCurrentPage(1);
   }
 
-  const paginationProps = {
+  const paginationTexts: PaginationTexts = {
+    pageSizeLabel: 'Rows per page:',
+    totalRowsText: 'Total number of rows:',
+    nextButtonAriaLabel: 'Next',
+    previousButtonAriaLabel: 'Previous',
+    numberButtonAriaLabel: (num) => `Page ${num}`,
```

Comment:
Not sure, but maybe we could use variables/types for these texts also? As done in Expression

Reply:
I can hardcode the texts for the tests, but I like having them in the stories file, so that they can be displayed in the Storybook example. This way, it becomes very clear what texts should be supplied to the component 😊 

See the code preview below:
![image](https://github.com/Altinn/altinn-studio/assets/148075168/f804ffd3-1e9c-4a11-bb8c-240ae9e727c1)